src/styles/_checkboxKnappWrapper.less

```diff
@@ -1,4 +1,4 @@
-.vurderBehovKnapp {
+.checkboxKnappWrapper {
```

Comment:
Siden du lager ny komponent så kanskje man kan bruke styled-component i stedet for less også?